### PR TITLE
Adding support for TAN67 PCB

### PR DESCRIPTION
**Pull request by Pangorin**
## Description
Adding VIA keymap support for TAN67 PCB

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/21003

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Review by Pangorin — COMMENTED** [review]
Layouts

**Comment by Xelus22** [comment]
I believe these 3 keys are arrows? Should be white?
image

**Review by Pangorin — COMMENTED** [review]
Fix the layout.

**Comment by Pangorin** [comment]
> I believe these 3 keys are arrows? Should be white? image

Yeah, I edited it!

**Comment by Xelus22** [comment]
Pleease check, build error

File: v3/pangorin/tan67/tan67.json [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        "4,1",
+        {"w": 1.25},
+        "4,2",
+        {"c": "#777777", "w": 6.25},
+        "4,5",
+        {"c": "#aaaaaa", "w": 1.25},
+        "4,9",
+        {"w": 1.25},
+        "4,10",
+        {"x": 0.5, c:"#cccccc"},
```
Comment (Xelus22) [inline v3/pangorin/tan67/tan67.json:105]:
```suggestion
        {"x": 0.5, "c":"#cccccc"},
```

**Review by Xelus22 — CHANGES_REQUESTED** [review]

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**